### PR TITLE
better scaling

### PR DESCRIPTION
**Pull request by jdroenner**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by jdroenner** [comment]
[Bessere Skalierung](https://trello.com/c/6qX1Hnno/1252-bessere-skalierung)

File: operators/src/processing/raster_scaling.rs [hunk 1]
```diff
@@ -25,26 +26,33 @@ use tracing::{span, Level};
...
-    Scale,
-    Unscale,
+    MulSlopeAddOffset,
+    SubOffsetDivSlope,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase", tag = "type")]
 enum PropertiesKeyOrValue {
+    Auto,
```
Comment (michaelmattig) [inline operators/src/processing/raster_scaling.rs:45]: the enum name implies that there are only two variants. Maybe name if `ScaleOffsetSelection`?

Comment (jdroenner) [reply operators/src/processing/raster_scaling.rs:45]: [4c22426](https://github.com/geo-engine/geoengine/pull/757/commits/4c224264049253672d4e61c0e0a5f36dc10a54dd)

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**